nipype/interfaces/fsl/ICA_AROMA.py

```diff
@@ -0,0 +1,106 @@
+                   desc='path/name of the mat-file describing the '
+                   'affine registration (e.g. FSL FLIRT) of the '
+                   'functional data to structural space (.mat file)')
+    fnirt_warp_file = File(exists=True, mandatory=True,
+                          argstr='-warp %s', xor=['feat_dir'],
+                          desc='File name of the warp-file describing '
+                          'the non-linear registration (e.g. FSL FNIRT) '
+                          'of the structural data to MNI152 space (.nii.gz)')
+    motion_parameters = File(exists=True, mandatory=True,
+                            argstr='-mc %s', xor=['feat_dir'],
+                            desc='motion parameters file')
+    denoise_type = traits.Enum('nonaggr', 'aggr', 'both', 'no', usedefault=True,
+                              mandatory=True, argstr='-den %s',
+                              desc='Type of denoising strategy: '
+                              '-none: only classification, no denoising '
+                              '-nonaggr (default): non-aggresssive denoising, i.e. partial component regression '
+                              '-aggr: aggressive denoising, i.e. full component regression '
+                              '-both: both aggressive and non-aggressive denoising (two outputs)')
+
+class ICA_AROMAOutputSpec(TraitedSpec):
+    aggr_denoised_file=File(exists=True, 
+                           desc='if generated: aggressively denoised volume')
+    nonaggr_denoised_file=File(exists=True,
+                              desc='if generated: non aggressively denoised volume' )
+    out_dir = Directory(exists=True, 
+                       desc='directory contains (in addition to the denoised files): '
+                       'melodic.ica + classified_motion_components + '
+                       'classification_overview + feature_scores + melodic_ic_mni)')
+   
+class ICA_AROMA(CommandLine):
+    """
+
+    Example
+    -------
+
+    >>> from nipype.interfaces.fsl import ICA_AROMA 
+    >>> from nipype.testing import example_data
+    >>> AROMA_obj = ICA_AROMA.ICA_AROMA()
+    >>> AROMA_obj.inputs.in_file=example_data('functional.nii')
+    >>> AROMA_obj.inputs.mat_file=example_data('func_to_struct.mat')
+    >>> AROMA_obj.inputs.fnirt_warp_file=example_data('warpfield.nii')
+    >>> AROMA_obj.inputs.motion_parameters=example_data('functional.par')
+    >>> AROMA_obj.inputs.mask=example_data('mask.nii.gz')
+    >>> AROMA_obj.inputs.denoise_type='both'
+    >>> AROMA_obj.inputs.out_dir='ICA_testout'
+    >>> AROMA_obj.cmdline
+    u'ICA_AROMA.py -den both -warp /home/travis/build/nipy/nipype/nipype/testing/data/warpfield.nii -i /home/travis/build/nipy/nipype/nipype/testing/data/functional.nii -m /home/travis/build/nipy/nipype/nipype/testing/data/mask.nii.gz -affmat /home/travis/build/nipy/nipype/nipype/testing/data/func_to_struct.mat -mc /home/travis/build/nipy/nipype/nipype/testing/data/functional.par -o ICA_testout'
```

Comment:
You don't need `example_data()`, which adds the full path, for doctests.

How about this:

```Python
>>> AROMA_obj.inputs.in_file = 'functional.nii'
>>> AROMA_obj.inputs.mat_file = 'func_to_struct.mat'
>>> AROMA_obj.inputs.fnirt_warp_file = 'warpfield.nii'
>>> AROMA_obj.inputs.motion_parameters = 'fsl_mcflirt_movpar.txt'
>>> AROMA_obj.inputs.mask = 'mask.nii.gz'
>>> AROMA_obj.inputs.denoise_type = 'both'
>>> AROMA_obj.inputs.out_dir='ICA_testout'
>>> AROMA_obj.cmdline # doctest: +ALLOW_UNICODE
'ICA_AROMA.py -den both -warp warpfield.nii -i functional.nii -m mask.nii.gz \
-affmat func_to_struct.mat -mc fsl_mcflirt_movpar.txt -o ICA_testout'
```